scalahost/nsc/src/test/scala/scala/meta/tests/scalahost/mirrors/SemanticSuite.scala

```diff
@@ -528,4 +528,16 @@ class SemanticSuite extends DatabaseSuite(SemanticdbMode.Slim) {
     }
   )
 
+  targeted(
+    """
+    object isJavaDefined {
```

Comment:
We really need to come up with a way to generate unique ids for top-level names in these tests.